authentik/enterprise/audit/apps.py

```diff
@@ -0,0 +1,19 @@
+"""Enterprise app config"""
+from django.conf import settings
+
+from authentik.enterprise.apps import EnterpriseConfig
+
+
+class AuthentikEnterpriseAuditConfig(EnterpriseConfig):
+    """Enterprise app config"""
+
+    name = "authentik.enterprise.audit"
+    label = "authentik_enterprise_audit"
```

Comment:
Did we want to stick with a label w/o `enterprise` in it in the event we move this to the OSS version?